app/javascript/react/components/PizzaShowContainer.js

```diff
@@ -49,6 +49,11 @@ const PizzaShowContainer = (props) => {
       },
       body: JSON.stringify(formPayload),
     })
+    if (!response) {
```

Comment:
might want to have `if (!response.ok)` otherwise you're saying if there isn't a response.

Reply:
I also just noticed this on line 23 as well... could you just double check for me. Add a debugger inside this block and send back something that should trigger it to make sure the sad path works.